MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -5746,28 +5759,44 @@ public int totalBonusParts() {
         return retVal;
     }
 
-    private int findAtBPartsAvailabilityLevel(IAcquisitionWork acquisition) {
-        AtBContract contract = getAttachedAtBContract(acquisition.getUnit());
+    public int findAtBPartsAvailabilityLevel(IAcquisitionWork acquisition, StringBuilder reportBuilder) {
+        AtBContract contract = acquisition != null ? getAttachedAtBContract(acquisition.getUnit()) : null;
         /*
          * If the unit is not assigned to a contract, use the least restrictive active
-         * contract
+         * contract. Don't restrict parts availability by contract if it has not started.
          */
         for (Mission m : getMissions()) {
-            if (m.isActive() && m instanceof AtBContract) {
+            if (m.isActive() && m instanceof AtBContract && ((AtBContract) m).getStartDate().isBefore(currentDay)) {
```

Comment:
This could be simplified and its performance can be boosted by swapping to an if statement checking hasActiveContract() followed by using getActiveContracts()